application/comit_node/src/swap_protocols/rfc003/ethereum/erc20_htlc.rs

```diff
@@ -33,21 +33,26 @@ impl Erc20Htlc {
     const CONTRACT_LENGTH_PLACEHOLDER: &'static str = "2002";
 
     pub fn new(
-        refund_timeout: Seconds,
+        refund_timestamp: Timestamp,
         refund_address: Address,
         redeem_address: Address,
         secret_hash: SecretHash,
         token_contract_address: Address,
         amount: Erc20Quantity,
     ) -> Self {
-        Self {
-            refund_timeout,
+        let htlc = Self {
+            refund_timestamp,
             refund_address,
             redeem_address,
             secret_hash,
             token_contract_address,
             amount,
-        }
+        };
+
+        trace!("Created new ERC20 HTLC for ethereum: {:#?}", htlc);
```

Comment:
I think #690 PR was too much removal but did not have time to comment on it.
I think we should log it once.

Reply:
Have you read the justification for the removal of the log statements?

"Once" is difficult to define in this case because the method is called *several* times, hence the duplicated log messages in the output.

Reply:
I think #690's PR was merged after I started with these tickets, so it just looks like I re-added it randomly, but it was there when I started working on this!